ui/client/components/graph/node-modal/subprocess-input-definition/MovableRow.tsx

```diff
@@ -0,0 +1,101 @@
+import {css, cx} from "emotion"
+import React, {PropsWithChildren, useRef} from "react"
+import {useDrag, useDrop} from "react-dnd"
+import {ReactComponent as Handlebars} from "../../../../assets/img/handlebars.svg"
+
+const TYPE = "field"
+
+interface MovableRowProps {
+  index: number,
+  moveItem: (a: number, b: number) => void,
+}
+
+interface DragItem {
+  index: number,
+  type: string,
+}
+
+export function MovableRow(props: PropsWithChildren<MovableRowProps>): JSX.Element {
```

Comment:
it would be ideal if we would add some new cypress tests to check this features - I mean in new PR